src/components/app/Home.js

```diff
@@ -10,26 +10,26 @@ import { AppHeader } from './AppHeader';
 import { InnerNavigationLink } from 'firefox-profiler/components/shared/InnerNavigationLink';
 import { ListOfPublishedProfiles } from './ListOfPublishedProfiles';
 
-import explicitConnect from '../../utils/connect';
+import explicitConnect from 'firefox-profiler/utils/connect';
 import classNames from 'classnames';
 import AddonScreenshot from '../../../res/img/jpg/gecko-profiler-screenshot-2019-02-05.jpg';
 import PerfScreenshot from '../../../res/img/jpg/perf-screenshot-2019-02-05.jpg';
 import FirefoxPopupScreenshot from '../../../res/img/jpg/firefox-profiler-button-2019-12-09.jpg';
```

Comment:
Ah, interesting! We should maybe create another alias for `./res` directory! (Not for this PR of course, it's not possible to use `firefox-profiler` for them.)